src/shares/version.ts

```diff
@@ -1,8 +1,20 @@
-import type VERSIONS from '../../versions.json';
+import VERSIONS from '../../versions.json';
 
 export type Version = (typeof VERSIONS)[number] & 'current';
 
 export function determinateVersionId(docVersion) {
 	if (!docVersion || !docVersion.preferredVersion) return null;
 	return parseFloat(docVersion?.preferredVersion?.name as Version) < 2 ? 'v1' : 'v2';
 }
+
+export function getVersionForUrl(docVersion: { preferredVersion?: { name: string } }) {
```

Comment:
1) Ich  würde das Parameter auf den `versionName` reduzieren, mehr wird in der Funktion nicht gebraucht.

2) Wir hatten auch noch besprochen, einen Kommentar hinzuzufügen, der erklärt, dass diese Funktion genutzt wird, um die Version so zurückzugeben, wie sie Docusaurus in der URL verwendet, und das der Rückgabewert deshalb auch ein leerer String sein kann. (Das ist jetzt mit `null` umgesetzt, passt für mich auch.)

3) Es wäre gut, mit TypeScript auch den Rückgabewert zu definieren. Dann müsste der Type nicht [wie hier](https://github.com/public-ui/public-ui.github.io/blob/64e3b9667d8e785209f87f4809467e58259cc6a3/src/components/HomepageFeatures.tsx#L30) überall bei den Funktionsaufrufen wiederholt werden.